fiat-core/src/main/java/com/netflix/spinnaker/fiat/util/RetrofitUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2025 OpsMx, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.fiat.util;
+
+import okhttp3.HttpUrl;
+
+public class RetrofitUtils {
```

Comment:
Seems like this belongs in kork.  We've got copies of this sprinkled around a few repos, right?  Happy to get these fixes in for now, but let's plan to centralize.